zone_future.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2020 The Inet.Af AUTHORS. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build go1.19
```

Comment:
Same question re 1.19 vs. 1.17. Do we feel okay that the runtime won't mutate too much until summer 2022?